external-crates/move/crates/move-analyzer/trace-adapter/src/adapter.ts

```diff
@@ -320,8 +320,16 @@ export class MoveDebugSession extends LoggingDebugSession {
                     stackFrames.push(extEventFrame);
                 }
             }
+            const stackFramesReversed = stackFrames.reverse();
```

Comment:
This is unrelated but since this PR is rather short, I thought I'd squeeze this one in here rather than creating a separate PR